test/libsolidity/util/BytesUtils.h

```diff
@@ -98,6 +102,11 @@ class BytesUtils
 		return formatString(_bytes, _bytes.size());
 	}
 
+	/// Converts \param _bytes to a soltest-compliant and human-readable
+	/// string representation of a byte array which is assumed to hold
+	/// a hexString value.
```

Comment:
Is this true? What does it mean? :)

Reply:
The comments where not yet updated, I just copied them. :D